eis_toolkit/raster_processing/distance_to_anomaly.py

```diff
@@ -0,0 +1,200 @@
+from itertools import chain
+from numbers import Number
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import geopandas as gpd
+import numpy as np
+import rasterio
+from beartype import beartype
+from beartype.typing import Literal, Tuple, Union
+from rasterio import profiles, transform
+
+from eis_toolkit.exceptions import InvalidParameterValueException
+from eis_toolkit.utilities.miscellaneous import row_points, toggle_gdal_exceptions
+from eis_toolkit.vector_processing.distance_computation import distance_computation
+
+THRESHOLD_CRITERIA_VALUE_TYPE = Union[Tuple[Number, Number], Number]
+THRESHOLD_CRITERIA_TYPE = Literal["lower", "higher", "in_between", "outside"]
+
+
+@beartype
+def distance_to_anomaly(
+    raster_profile: Union[profiles.Profile, dict],
+    anomaly_raster_profile: Union[profiles.Profile, dict],
+    anomaly_raster_data: np.ndarray,
+    threshold_criteria_value: THRESHOLD_CRITERIA_VALUE_TYPE,
+    threshold_criteria: THRESHOLD_CRITERIA_TYPE,
+) -> np.ndarray:
+    """Calculate distance from raster cell to nearest anomaly.
+
+    The criteria for what is anomalous can be defined as a single number and
+    criteria text of "higher" or "lower". Alternatively, the definition can be
+    a range where values inside (criteria text of "within") or outside are
+    marked as anomalous (criteria text of "outside").
+
+    Args:
+        raster_profile: The raster profile of which the distances
+            to the nearest anomalous value are determined.
+        anomaly_raster: The raster in which the distances
+            to the nearest anomalous value are determined.
```

Comment:
In the parameters, `anomaly_raster_profile` and `anomaly_raster_data` are separate. I suppose you just forgot to update the docs here?

Reply:
Yep, updating!